src/server/pages/index.js

```diff
@@ -10,7 +10,23 @@ export default function renderPage (req, res) {
 
   res.setHeader('Content-Type', 'text/html')
   function send (html) {
-    res.send(`<!DOCTYPE html><head><style type="text/css">${css()}</style><meta name="viewport" content="width=device-width, initial-scale=1"></head><body class="__next-auth-theme-${theme}"><div class="page">${html}</div></body></html>`)
+    res.send(`<!DOCTYPE html>
+      <html lang="en">
+      <head>
+          <meta charset="UTF-8">
+          <meta http-equiv="X-UA-Compatible" content="IE=edge">
+          <meta name="viewport" content="width=device-width, initial-scale=1.0">
+          <style>
+              ${css()}
+          </style>
+          <title>Next Auth Sign in</title>
```

Comment:
This one is probably a good idea to add, but if you check the lines 33-36 in this file, you can see that simply setting `Next Auth Sign in` is not sufficient. I think a nice way of setting a custom title for each of the page types would be if the `send` function took a `title` param in addition to html, and add individual titles for `signin`, `signout`, `verifyRequest` and `error`.

I would also prefer if the function signature becomes: `function send({html, title})` to future proof us. #1154 would require us to send `locale` or something similar here.

Reply:
Ok, so should I change the function's signature to `function send({ html, title })`?

Reply:
yes. Usually I have a rule of thumb, if you need more than one param I just like to name them for convinience.